include/ultra64/rcp.h

```diff
@@ -136,4 +154,21 @@
 #define VI_PAL_CLOCK            49656530    /* Hz = 49.656530 MHz */
 #define VI_MPAL_CLOCK           48628316    /* Hz = 48.628316 MHz */
 
+#define PI_DOM1_ADDR1   0x06000000  /* to 0x07FFFFFF */
+#define PI_DOM1_ADDR2   0x10000000  /* to 0x1FBFFFFF */
+#define PI_DOM1_ADDR3   0x1FD00000  /* to 0x7FFFFFFF */
+#define PI_DOM2_ADDR1   0x05000000  /* to 0x05FFFFFF */
+#define PI_DOM2_ADDR2   0x08000000  /* to 0x0FFFFFFF */
+
+/**
+ * External device info
+ */
+#define DEVICE_TYPE_CART    0   /* ROM cartridge */
+#define DEVICE_TYPE_BULK    1   /* ROM bulk */
+#define DEVICE_TYPE_64DD    2   /* 64 Disk Drive */
+#define DEVICE_TYPE_SRAM    3   /* SRAM */
+/* 4-6 are reserved */
+#define DEVICE_TYPE_INIT    7   /* initial value */
+/* 8-14 are reserved */
```

Comment:
Only thing I found on this:

> rcp.h
> Changed 8-14 for DEVICE_TYPE to reserve. 

(in "Release Notes OS2.0K" `n64man/relnote_k/index.htm`)

do you know more lol

Reply:
The comments are just lifted straight from OS2.0L rcp.h

Reply:
so rcp.htm in the manual is out-of-date and I guess others too
I'll flee those in the future